x-pack/plugins/apm/server/lib/services/get_service_instances/get_service_instances_system_metric_statistics.ts

```diff
@@ -68,7 +68,12 @@ export async function getServiceInstancesSystemMetricStatistics<
 }): Promise<Array<ServiceInstanceSystemMetricStatistics<T>>> {
   const { apmEventClient } = setup;
 
-  const { intervalString } = getBucketSize({ start, end, numBuckets });
+  const { intervalString } = getBucketSize({
+    start,
+    end,
+    numBuckets,
+    searchAggregatedTransactions: true,
```

Comment:
This does not query transaction metrics, just system metrics, which have a default collection interval of 30s IIRC.